src/main/java/com/openlattice/controllers/OrganizationsController.java

```diff
@@ -208,24 +233,62 @@ public OrganizationIntegrationAccount getOrganizationIntegrationAccount( @PathVa
     public Map<UUID, Set<OrganizationEntitySetFlag>> assembleEntitySets(
             @PathVariable( ID ) UUID organizationId,
             @RequestBody Set<UUID> entitySetIds ) {
+        final var authorizedPropertyTypesByEntitySet =
+                getAuthorizedPropertiesForMaterialization( organizationId, entitySetIds );
+        return assembler.materializeEntitySets( organizationId, authorizedPropertyTypesByEntitySet );
+    }
+
+    @Override
+    @PostMapping( ID_PATH + SET_ID_PATH + SYNCHRONIZE )
+    public Void synchronizeEdmChanges(
+            @PathVariable( ID ) UUID organizationId,
+            @PathVariable( SET_ID ) UUID entitySetId ) {
+        // we basically re-materialize in this case
+        final var authorizedPropertyTypesByEntitySet =
+                getAuthorizedPropertiesForMaterialization( organizationId, Set.of( entitySetId ) );
+
+        assembler.synchronizeMaterializedEntitySet(
+                organizationId,
+                entitySetId,
+                authorizedPropertyTypesByEntitySet.values().iterator().next());
```

Comment:
Why not pass `authorizedPropertyTypesByEntitySet.get( entitySetId )`? seems a little more straightforward / less likely to break if something changes

Reply:
Yeah, I don't know why I did that, lol